docs/targets.rst

```diff
@@ -53,3 +53,33 @@ by passing them to Actions::
         Enter.the_secret("reallySecurePassword!!").into_the(PASSWORD_FIELD),
         Click.on_the(SIGN_IN_BUTTON),
     )
+
+The resulting log::
+
+    Webster enters "webster_1987" into the username field.
+    Webster enters "[CENSORED]" into the password field.
+    Websert clicks on the "Sign In" button.
```

Comment:
Thank you for adding these! I know we talked about them months ago.

One thing to note though, Sphinx/RST treats `::` as meaning "the next block of code is Python". We might want to use a blockquote here instead. I think all you need to do is remove one of the colons, so:

```
The resulting log:

    Webster enters "webster_1987" into the username field.
    Webster enters "[CENSORED]" into the password field.
    Websert clicks on the "Sign In" button.
```

You can try it yourself by doing `pip install sphinx`, then `make clean; make html` in the `docs` folder. After that you can open the `_build/html/index.html` file in any browser and navigate to the Target page. It will probably have a different style because the built-in styles don't include the one i've selected, just ignore that.